src/Microsoft.Health.Fhir.Shared.Api/Controllers/ExportController.cs

```diff
@@ -77,32 +77,25 @@ public class ExportController : Controller
         [AuditEventType(AuditEventSubType.Export)]
         public async Task<IActionResult> Export([FromQuery(Name = KnownQueryParameterNames.Since)] PartialDateTime since)
         {
-            if (!_exportConfig.Enabled)
-            {
-                throw new RequestNotValidException(string.Format(Resources.OperationNotEnabled, OperationsConstants.Export));
-            }
-
-            CreateExportResponse response = await _mediator.ExportAsync(_fhirRequestContextAccessor.FhirRequestContext.Uri, since, HttpContext.RequestAborted);
-
-            var exportResult = ExportResult.Accepted();
-            exportResult.SetContentLocationHeader(_urlResolver, OperationsConstants.Export, response.JobId);
-
-            return exportResult;
+            CheckIfExportIsEnabled();
+            return await CreateExportJob(since);
         }
 
         [HttpGet]
         [Route(KnownRoutes.ExportResourceType)]
         [ServiceFilter(typeof(ValidateExportRequestFilterAttribute))]
         [AuditEventType(AuditEventSubType.Export)]
-        public IActionResult ExportResourceType(string typeParameter)
+        public async Task<IActionResult> ExportResourceType([FromQuery(Name = KnownQueryParameterNames.Since)] PartialDateTime since, string typeParameter)
         {
+            CheckIfExportIsEnabled();
+
             // Export by ResourceType is supported only for Patient resource type.
```

Comment:
Curious, why other resource types are not supported?

Reply:
FHIR spec :)

Reply:
The FHIR spec only says Patient types should be supported.
https://hl7.org/fhir/uv/bulkdata/export/index.html#endpoint---all-patients

The spec doesn't say the other types shouldn't be supported, but it only lists Patient as an export operation.
If we remove this check we would support any resource type, so their is no technical lift to add that. It just a question of if we are still following the spec if we add that.